packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -1082,6 +1088,32 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
     _syncAll(widget.gestures);
   }
 
+  @override
+  void didChangeAccessibilityFeatures() {
+    if (!widget.excludeFromSemantics)
```

Comment:
If we're excluded, should we just not listen add us as a WidgetsBindingObserver?